ammo-fe/cypress/integration/TopBar.cy.ts

```diff
@@ -0,0 +1,65 @@
+import WS from '../../src/network/WS';
+import { bulletMock } from '../../src/shared/mocks/Bullet.mock';
+
+const socket = WS.getSocket('http://localhost:3001');
+
+describe('Top bar', () => {
+    before(() => {
+        cy.visit('/');
+    });
+
+    it('Recorder button is clickable as we are connected', () => {
+        cy.get('[data-cy=recorder-button]').should('be.enabled');
+    });
+
+    it('Recorder button first image must be pause', () => {
+        cy.get('[data-cy=recording-button-pause]').should('not.be.visible');
+    });
+
+    it('Recorder button must switch inner images on toggle', () => {
+        cy.get('[data-cy=recorder-button]').click();
+
+        cy.get('[data-cy=recording-button-record]').should('be.visible');
+
+        cy.get('[data-cy=recording-button-pause]').should('not.be.visible');
+
+        cy.get('[data-cy=recorder-button]').click();
+
+        cy.get('[data-cy=recording-button-record]').should('not.be.visible');
+
+        cy.get('[data-cy=recording-button-pause]').should('be.visible');
+    });
+
+    it('Pause the recorder and emit a bullet - Must not appear in the UI', () => {
```

Comment:
Prefer 'should' to 'must' as we don't know if the tests will fail. Otherwise excellent namings & tests